### DISCUSSION TITLE
Trigger email when item of an order shipped with Order # and other details of order to the customer

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#1689] [Q&A]
I have to send an email when an item of order is shipped. I came across '**FulfillmentStateTransitionEvent**', which I can hook for events like Shipped, Delivered, etc. As part of the email, I am supposed to send the Order Number and customer on the Order to fetch the recipient's email. How can I retrieve this information from Fulfilment?

**Comment by michaelbromley** [comment]
Hi,
You can use the [loadData method](https://www.vendure.io/docs/typescript-api/email-plugin/email-event-handler/#loaddata) on the EmailEventHandler to perform any kind of data lookup from the database.

Also, you might also want to consider subscribing to the `OrderStateTransitionEvent` which should also be published when the Order transitions to `Delivered`, and that will already include a reference to the order.
